### PR TITLE
square function

### PR DESCRIPTION
**Pull request by dfarr**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/reviewninja/foo/pull/5" target="_blank">ReviewNinja

**Comment by review-ninja** [comment]
<a href="https://staging.review.ninja/reviewninja/foo/pull/5" target="_blank">ReviewNinja

File: math.js [hunk 1]
```diff
@@ -6,6 +6,10 @@ module.exports = {
   
   sub: function(a, b) {
     return a - b;
+  },
+  
+  square: function(a) {
```
Comment (dfarr) [inline math.js]: test bro

Comment (dfarr) [inline math.js]: test bro

Comment (dfarr) [inline math.js]: and another...

Comment (dfarr) [inline math.js]: and again...

Comment (dfarr) [inline math.js]: another

Comment (dfarr) [inline math.js]: **more**

File: math.js [hunk 2]
```diff
@@ -6,6 +6,10 @@ module.exports = {
```
Comment (dfarr) [inline math.js]: another thing...

Comment (dfarr) [inline math.js]: !fix

Comment (dfarr) [inline math.js]: !fixed

Comment (dfarr) [inline math.js]: !fix

Comment (dfarr) [inline math.js]: !fixed

Comment (dfarr) [inline math.js]: !fix

Comment (dfarr) [inline math.js]: !fixed

Comment (dfarr) [inline math.js]: !fix

Comment (dfarr) [inline math.js]: !fixed

**Pull request merged**